a/airline/airline_rhel7.6.sh

```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package			: airline
+# Version			: 0.9
+# Source repo		: https://github.com/airlift/airline
+# Tested on			: RHEL 7.6
+# Script License	: Apache License Version 2.0
+# Maintainer		: Vedang Wartikar <vedang.wartikar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+# install tools and dependent packages
+yum update -y
+yum install git -y
+
+# setup java environment
+yum install java java-devel -y
+JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk"
+echo $JAVA_HOME
+export PATH=$PATH:$JAVA_HOME/bin
+java -version
+
+# install maven
+dnf install maven -y

```

Comment:
why are we mixing yum and dnf commands? which base was this tested on? RHEL/UBI/UBI minimal?

Reply:
this was initially tested on a centos container. didn't have a RHEL VM when I was creating this build script